Sources/OpenAPIURLSession/URLSessionBidirectionalStreaming/HTTPBodyOutputStreamBridge.swift

```diff
@@ -0,0 +1,286 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftOpenAPIGenerator open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftOpenAPIGenerator project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftOpenAPIGenerator project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+import OpenAPIRuntime
+import HTTPTypes
+#if canImport(Darwin)
+import Foundation
+
+final class HTTPBodyOutputStreamBridge: NSObject, StreamDelegate {
+    static let streamQueue = DispatchQueue(label: "HTTPBodyStreamDelegate", autoreleaseFrequency: .workItem)
+
+    let httpBody: HTTPBody
+    let outputStream: OutputStream
+    private(set) var state: State {
+        didSet { debug("Output stream delegate state transition: \(oldValue) -> \(state)") }
+    }
+
+    init(_ outputStream: OutputStream, _ httpBody: HTTPBody, openOutputStream: Bool = true) {
```

Comment:
Is `openOutputStream` still useful? I think it's always true now

Reply:
I think we _used_ to _not_ open it in tests where we wanted some finer grain control, but I can see that you're right: it is always true now—will remove, thanks!

Reply:
Removed `openOutputStream` initializer parameter in 9bb44eb.